install/update_91_92.php

```diff
@@ -633,6 +635,75 @@ function update91to92() {
    Config::setConfigurationValues('core', array('login_remember_time'    => 604800,
                                                 'login_remember_default' => 1));
 
+   if (TableExists('glpi_bookmarks')) {
+      $migration->renameTable("glpi_bookmarks", "glpi_savedsearches");
+
+      $migration->addField("glpi_savedsearches", "last_execution_time", "int(11) NULL DEFAULT NULL");
+      $migration->addKey("glpi_savedsearches", 'last_execution_time');
+
+      $migration->addField("glpi_savedsearches", "do_count", "tinyint(1) NOT NULL DEFAULT '2' COMMENT 'Do or do not count results on list display; see SavedSearch::COUNT_* constants'");
+      $migration->addKey("glpi_savedsearches", 'do_count');
+
+      //ensure do_count is set to AUTO
+      $migration->addPostQuery('UPDATE glpi_savedsearches SET do_count = ' . SavedSearch::COUNT_AUTO);
+      $migration->addPostQuery('UPDATE glpi_savedsearches SET entities_id = 0 WHERE entities_id = -1');
+   }
+
+   if (TableExists('glpi_bookmarks_users')) {
+      $migration->renameTable("glpi_bookmarks_users", "glpi_savedsearches_users");
+      $migration->changeField('glpi_savedsearches_users', 'bookmarks_id', 'savedsearches_id', 'int(11) NOT NULL DEFAULT "0"');
+   }
+
+   //TRANS: %s is the table or item to migrate
+   $migration->displayMessage(sprintf(__('Data migration - %s'), 'glpi_displaypreferences'));
+
+   $ADDTODISPLAYPREF['SavedSearch'] = array(8, 9, 3, 10, 11);
```

Comment:
if we need this, we could check simply presence of lines for users_id = 0 (still simpler than the current code) or do REPLACE INTO instead INSERT INTO (the unique index will choose the add or update lines)